Scripts/CLI.py

```diff
@@ -1,31 +1,39 @@
 """
-This is the CLI module. More documentation to come.
+This is the CLI module.
+
+This is the input/output module for the system. It allows a user to
+input required data (i.e. spice kernel file, asteroid image, shape, and execution time)
+and generates a poly-line (list of coordinates) as output.
 """
 # Import packages/modules here
 import click
 
-
 @click.command()
-@click.option('--count', default=1, help='Number of executions.')
-@click.option('--spice_kernel', prompt='Spice kernel file: ', help='The file path of the spice kernel')
-@click.option('--shape', default='Irregular', help='The configuration \
-of the shape for the model')
-def main(count, spice_kernel, shape):
+@click.option('--spice-kernel', prompt='Spice kernel file', help='The file path of the spice kernel')
```

Comment:
We’ll probably need a separate option for each type of kernel file. @brianvu155 @purplecow400 might know which ones would be required. It’s not a critical change to make right now though.

Reply:
Okay should I not do anything for the time being?

Reply:
Just add a `# TODO:` comment so we know to come back to this.